### DISCUSSION TITLE
golangci-lint github actions seems not reading config file

### DISCUSSION THREAD

**Discussion by ThiamSoon** [golangci/golangci-lint#2142] [Q&A]
Hi,

I'm running golangci/golangci-lint-action@v2.5.2 githubactions. From the log it seems it does not read the config file in my root directory. I set output format to checkstyle but it still output as github actions. Can you help to advice?

Thanks.

**Comment by Antonboom** [comment]
@ThiamSoon, hi!
Still actual for `golangci/golangci-lint-action@v3`?
